src/kafka/encode_decode.go

```diff
@@ -38,6 +38,16 @@ func (d *Decoder) DeserializerFor(recordType string) DecodeMessageFunc {
 }
 
 func (d *Decoder) AvroMessageToRecord(context context.Context, msg *sarama.ConsumerMessage) (*models.Record, error) {
+	if msg.Value == nil {
```

Comment:
shouldn't we return an error here?

Reply:
if i return an error, it will try to consume the message again.

Reply:
We can return a specific error(`ErrNilMessage`), and check for it. Another option is to filter directly on the consumer.

Reply:
ok, i will do it